test/inductor/test_torchinductor.py

```diff
@@ -1745,6 +1745,38 @@ def fn(a, b):
             check_lowp=False,
         )
 
+    @config.patch(use_mixed_mm=True)
+    def test_mixed_mm(self):
+        def fn(a, b):
+            return torch.mm(a, b.to(a.dtype))
+
+        if self.device == "cuda":
+            self.common(
+                fn,
+                (
+                    torch.randn(8, 8),
+                    torch.randint(-128, 127, (8, 8), dtype=torch.int8),
+                ),
+                check_lowp=True,
+            )
+
+    @config.patch(use_mixed_mm=True)
+    def test_mixed_mm2(self):
+        def fn(a, b, scale, bias):
+            return torch.mm(a, b.to(a.dtype)) * scale + bias
+
+        if self.device == "cuda":
+            self.common(
+                fn,
+                (
+                    torch.randn(8, 8),
+                    torch.randint(-128, 127, (8, 8), dtype=torch.int8),
+                    torch.randn(8),
+                    torch.randn(8),
+                ),
+                check_lowp=True,
+            )
+
```

Comment:
We should include a test that the kernel/code is actually being used.  See test_select_algorithm for examples.

Reply:
there is a test in the pattern matcher that checks it, it seems like it checks the same thing, right?